src/containers/Splash.js

```diff
@@ -16,7 +16,7 @@ const Splash = (props) => (
     <Content className="container jumbotron">
       <Title>FIZCA</Title>
       <p>It means soul in muisca!</p>
-      <p><a className='btn fz-btn-dark' href={`http://${ServerUrl}/google/oauth`}>Login</a></p>
+      <p><a className='btn fz-btn-dark' href={`${ServerUrl}/google/oauth`}>Login</a></p>
     </Content>
```

Comment:
Yay for removing http://, can the rest of this be constructed before, or is it sort of assumed this is how urls are made?

Reply:
We can probably defined things differently. Like some BackendApi class that knows what to call when. I'm very much opened to suggestions.